course_flow/views.py

```diff
@@ -550,11 +547,11 @@ def duplicate_course(course: Course, author: User) -> Course:
         is_original=False,
         parent_course=course,
```

Comment:
Change to parent_workflow=workflow and reference the workflow object of the activity, rather than the activity itself.